rapids-cmake/export/export.cmake

```diff
@@ -154,7 +155,7 @@ function(rapids_export type project_name)
   string(TOLOWER ${project_name} project_name)
   string(TOUPPER ${project_name} project_name_uppercase)
   if(type STREQUAL "install")
-    set(install_location "${CMAKE_INSTALL_LIBDIR}/cmake/${project_name}")
+    set(install_location "lib/cmake/${project_name}")
```

Comment:
Could we maybe do something like?
```
if(${CMAKE_INSTALL_PREFIX} EQUAL $ENV{CONDA_PREFIX})
    set(install_location "lib/cmake/${project_name}")
else()
    set(install_location "${CMAKE_INSTALL_LIBDIR}/cmake/${project_name}")
endif()
```

I hate to make things conda specific when we don't need to.

Reply:
I agree, I do want to stick with presuming always a conda environment. 

For `21.08` I want to stick with the minimal change required to fix conda, and I think the current change is the correct approach. While yours is better, we still have the issue that numerous RAPIDS packages hard code the install location of things like `libcudf` to always be `lib` instead of using to GNUInstallDirs. 

For `21.10` we should have a better holistic approach for RAPIDS. This would mean providing a function like `rapids_cmake_install_location` that would compute either `lib` or `lib64` based on CONDA, etc. That way not only is `rapids_export` fixed, but we can also update cudf, cugraph, cuml, ....

Reply:
Once we/I figure out a robust approach that we are doing a conda install I will upstream it to CMake